src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
 export class Triangle {
+  shape: Shape = 'triangle';
```

Comment:
all figures class must implement class Figure 
```suggestion
export class Triangle implements Figure {
  shape = 'triangle';
```

Reply:
shape won`t be working if i will remove type